src/Home/PaletteEditor/PaletteGod.ZC

```diff
@@ -1,15 +1,23 @@
+CBGR24 ColorGod(Bool instant=TRUE)
+{
+ 	// Generate a random color
+ 	if (instant)
+ 		return RandI32 & 0xFFFFFF;
+ 	else
+ 		return GodPick & 0xFFFFFF;
+}
 
-public U0 PaletteGodGenerate()
+public U0 PaletteGod(Bool seedUse=FALSE, Bool instant=TRUE)
 {
+ 	// Generate a random palette, seed-based or not
 	I64		i;
-	CBGR24	palette;
 
-	for (i = 0; i < 16; i++)
-	{
-		palette = GodPick & 0xFFFFFF;
-		gr_palette[i] = palette;
-		LFBFlush;
+	if (seedUse) {
+		Seed(GodPick);
 	}
```

Comment:
You can make this 2 lines instead of 3.

Reply:
@xslendix you can actually make this one line trollface.jpg
`if (seedUse) Seed(GodPick);` 

Thanks for the review, I'll work on it this weekend.